prototype2/server/src/engine/mod.rs

```diff
@@ -1,120 +1,57 @@
-mod event_handler;
-mod physics;
+mod neo_physics;
+mod neo_network;
+mod neo_player;
+mod neo_debug;
```

Comment:
I'm actually not going to abc: the work to maintain the abc isn't worth it: will wait for an editor plugin to do this for me.